tasks/gen/templates/model/{{name}}.cr.ecr

```diff
@@ -0,0 +1,4 @@
+class <%= @name.capitalize %> < BaseModel
+  table :<%= @pluralized_name %> do
```

Comment:
TIL Crystal doesn't provide an option to convert a String to a Symbol, hence the weird interpolation here.